pandas/core/frame.py

```diff
@@ -887,15 +887,16 @@ def itertuples(self, index=True, name="Pandas"):
 
         Parameters
         ----------
-        index : boolean, default True
+        index : bool, default True
             If True, return the index as the first element of the tuple.
-        name : string, default "Pandas"
+        name : str, default "Pandas"
             The name of the returned namedtuples or None to return regular
             tuples.
 
-        Returns
+        Yields
         -------
-        Returns namedtuples.
+        collections.namedtuple
+            Yields namedtuples of corresponding index and column values.
```

Comment:
I see what you mean here, but I think it may be difficult to understand for a beginner. May be something like `Yields a namedtuple for each row in the DataFrame, which its values, and possibly the index.`.

Reply:
Updated the description just now! I think it's more intuitive now, please do let me know and if anything needs to be added I'll try again.